File: Adapter.java/src/com/company/EnemyAircraft.java
package com.company;

import java.util.Random;

public class EnemyAircraft implements EnemyAttacker{
    Random random = new Random();

    @Override
    public void shootBullet() {
        int attackDamage = random.nextInt(10) +1 ;
        System.out.println("Enemy Aircraft does " + attackDamage + " damages");
    }

    @Override
    public void driveForward() {
        int movement = random.nextInt(15) + 1;
        System.out.println("Enemy Aircraft moves " + movement + " spaces");
    }

    @Override
    public void assignDriver(String driverName) {
        System.out.println(driverName + " is driving the Aircraft");
    }
}

File: Adapter/EnemyAircraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Adapter {
    internal class EnemyAircraft : IEnemyAttacker {
        private readonly Random random = new();

        public void AssignDriver(string driverName) {
            Console.WriteLine(driverName + " is driving the Aircraft");
        }

        public void DriveForward() {
            int movement = random.Next(15) + 1;
            Console.WriteLine("Enemy Aircraft moves " + movement + " spaces");
        }

        public void ShootBullet() {
            int attackDamage = random.Next(10) + 1;
            Console.WriteLine("Enemy Aircraft does " + attackDamage + " damages");
        }
    }
}
